cookbooks/impala/recipes/packages.rb

```diff
@@ -13,7 +13,8 @@
 when "debian"
   Chef::Log.info('Debian family')
   case
-  when node['platform_version'] == '15.04', node['platform_version'] == '14.04'
+  when node['platform_version'] == "16.04", node['platform_version'] == "15.10",
```

Comment:
The chef script does not work with 16.04. So, I am not sure it's a good idea to include this here.